src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+  }
+  &__day:hover {
+    background-color: #ffbfcb;
```

Comment:
Similar to the previous comment about color literals, `#ffbfcb` should also be stored in a variable.